access-control-app/src/cmr/access_control/config.clj

```diff
@@ -54,6 +54,11 @@
   "Feature toggle to enable the GET verb for /current-sids - remove after all clients have been migrated"
   {:default true :type Boolean})
 
+;; TODO: This can be removed if no clients are found to need open access to ACLs.
+(defconfig lockdown-acl-get-requests
+  "Flag for whether or not launchpad token is required to access acl get requests."
+  {:default true :type Boolean})
+
```

Comment:
after some time, we should remove this setting and it should just be the norm, but we don't know what clients might break or who will complain, so I am adding a way for rolling this out slowly.